### PR TITLE
 Added code of daily Coding Exercise- 15

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/MatrixAddition.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public static void main(String args[]) {
+        int matrix[][] = {{1, 3, 4}, {2, 4, 3}, {3, 4, 5}};
+        int subMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {1, 2, 4}};
+        int result[][] = new int[3][3];
+        for (int index = 0; index < 3; index++) {
+            for (int subIndex = 0; subIndex < 3; subIndex++) {
+                result[index][subIndex] = matrix[index][subIndex] + subMatrix[index][subIndex];
+                System.out.print(result[index][subIndex] + " ");
+            }
+            //new line
```
Comment (AmrutaDhage20) [inline First/dailyExercises/MatrixAddition.java]: Remove the comment.

Comment (anushreeSonak) [reply First/dailyExercises/MatrixAddition.java]: Done!

**Review by AmrutaDhage20 — APPROVED** [review]

File: First/dailyExercises/MatrixAddition.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package firstPackage.First.dailyExercises;
+
+public class MatrixAddition {
+    public static void main(String args[]) {
+        int matrix[][] = {{1, 3, 4}, {2, 4, 3}, {3, 4, 5}};
+        int subMatrix[][] = {{1, 3, 4}, {2, 4, 3}, {1, 2, 4}};
+        int result[][] = new int[3][3];
+        for (int index = 0; index < 3; index++) {
```
Comment (Vrushalee1234) [inline First/dailyExercises/MatrixAddition.java]: Write code to accept n*n dimension. 3 seems to be hardcoded.

**Review by Vrushalee1234 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/MatrixAddition.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/MatrixAddition.java]: Done !!

**Review by Vrushalee1234 — APPROVED** [review]

**Pull request merged**